xdsl/frontend/dialects/arith.py

```diff
@@ -4,86 +4,86 @@
 from xdsl.frontend.dialects.builtin import index, i1, i32, i64, f16, f32, f64
 from xdsl.ir import Operation
 
-_Int = TypeVar("_Int", bound=Union[index, i1, i32, i64], covariant=True)
+Int = TypeVar("Int", bound=Union[index, i1, i32, i64])
```

Comment:
The type is used in the interface of a public function, feels like it should be visible to clients. But now that I look at it maybe, I can change it back. Will try that out rn